app/build.gradle

```diff
@@ -98,28 +98,30 @@ android {
 				'TrustAllX509TrustManager', // accept certificates as implemented in PTE
 				'GradleDependency', // don't complain about old library versions
 				'IconDensities', //todo: migrate all (network) icons to vectors
-				'PrivateResource', //todo: investigate about private Material design resources (e.g. CarcView style)
-				'Typos', //todo: remove after next translation fetch
-				'TypographyEllipsis' //todo: remove after next translation fetch
+				'PrivateResource' //todo: investigate about private Material design resources (e.g. CardView style)
 	}
 }
 
 dependencies {
 	implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
 
-	implementation('androidx.appcompat:appcompat:1.2.0-alpha03') {
-		because 'fixes bug where stopped activities would not resume'
+	implementation 'com.google.android.material:material:1.2.1'
+	implementation 'androidx.appcompat:appcompat:1.2.0'
+	implementation 'androidx.activity:activity:1.1.0'
+	implementation('androidx.fragment:fragment:1.1.0!!') {
+		because 'issues with MapBox SDK onDestroy after upgrading to 1.2.0-alpha02 or newer versions'
```

Comment:
Hopefully, this won't cause issues with other AndroidX libraries depending on this one.